docs/bcfusion/developers/staking.md

```diff
@@ -134,7 +134,105 @@ function claimBatch(address[] calldata operatorAddresses, uint256[] calldata req
 - `operatorAddress`: The operator addresses of the validatores.
 - `requestNumber`: The numbers of unbonding requests to claim from the validators.
 
+## FAQs
+
+### What are the functions/interfaces of each validator's credit contract?
+
+For each validator, there is a credit contract which will be automatically deployed when it is created.
+The credit contract is a BEP20 contract, and the ABI is the same
+as [Stake Credit contract](https://github.com/bnb-chain/bsc-genesis-contract/blob/master/abi/stakecredit.abi).
+
+Meanwhile, the conctract cannot be upgraded or changed by any validator operator.
+
+### How to get the shares/BNB amount for a delegator?
+
+For any specific validator, you can call the `balanceOf` function of the validator's creat contract to get the
+delegator's shares. If you want to get the BNB amount instead of shares, you can use `getPooledBNB` function.
+
+If you want to get the shares of all validators, you need to call the `balanceOf` function for each valiator and sum up
+the results. Please refer to the following to see how to get the information of all validators, and use muticall to
+improve the efficiency.
+
+### How to calculte the BNB amount for a specific amount of shares?
+
+The credit contract provides the `getPooledBNBByShares` function to calculate the BNB amount for some specific amount of
+shares.
+
+If you want to do the vice visa, you can use the `getSharesByPooledBNB` function to calculate the shares for some
+specific BNB amount.
+
+### How to calculte the APR/APY of a validator?
+
+Please be noted that each validator will have its own APR/APY, and the staking system will auto compound the rewards.
+
+The reward is distributed to each validator's BNB pool at 00:00:00 UTC time every day. To calculate the APR/APY of a
+validator, you need to get the total pooled BNB amount and the crrospanding reward amount for the same day.
+
+The `StakeHub` contract provides the `getValidatorTotalPooledBNBRecord(address,uint256)(uint256)`
+and `getValidatorRewardRecord(address,uint256)(uint256)` for the purpose.
+
+The following code shows how to get the total pooled BNB amount and the crrospanding reward amount for a given time:
```

Comment:
why not change "shows how to get the total pooled BNB amount and the crrospanding reward amount for a given time" ==> `shows how to calculate the APY at a given day`?

Reply:
fixed